parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -266,24 +266,24 @@ write_files:
     [plugins."io.containerd.grpc.v1.cri"]
       sandbox_image = "{{GetPodInfraContainerSpec}}"
       [plugins."io.containerd.grpc.v1.cri".containerd]
-        {{ if TeleportEnabled }}
+        {{- if TeleportEnabled }}
```

Comment:
why touch teleport here? Must admit I am not familiar with when - is needed or not